cli/metric/performance.go

```diff
@@ -88,7 +89,12 @@ func (f *PerformanceFlag) Interval(val int32) int32 {
 			if err != nil {
 				panic(err)
 			}
-			interval = int32(n)
+
+			if n > math.MaxInt32 {
+				panic(fmt.Errorf("value out of range for int32: %d", n))
```

Comment:
Since we're parsing user input here.. we should change `Interval()` to return `(int32, error)`, rather than the existing panic() and adding another. But this hasn't been raised as an issue before either, find to keep as-is.